site/docs/transform/filter.md

```diff
@@ -46,6 +46,54 @@ For example, to check if the `car_color` field's value is equal to `"red"`, we c
 {"filter": {"field": "car_color", "equal": "red"}}
 ```
 
+{:#lt-predicate}
+### Field Less Than Predicate
```

Comment:
@domoritz  I'm wondering if we should really add 4 sections for lt,lte,gt,gte.  To me, they are pretty much the same.  So adding 4 sections maybe a bit too much.  That said, may be it's fine.

Reply:
Let's have a single section.

Reply:
Ok.  We chatted and let's keep it this way.  There are many other things we should focus more on.

Reply:
Actually, let's name the section 

### Field Comparator Predicates

and simplify the content?

Reply:
(You can omit the tables and say that instead of `equal`, you can use `lt`, `gt`, ... for less than , greater than, ...)

Reply:
Should `equal` be a part of `Field Comparator Predicates`?